heron_base/launch/base.launch

```diff
@@ -32,23 +32,25 @@
     <group unless="$(optenv HERON_IMU_UM6 0)">
       <!-- newer herons use the microstrain CM5-25 IMU -->
       <group if="$(eval arg('namespace') == '')">
-        <include filename="$(find ros_mscl)/launch/microstrain.launch">
+        <include file="$(find ros_mscl)/launch/microstrain.launch">
           <arg name="name"          value="cv5" />
           <arg name="port"          value="$(optenv HERON_IMU_PORT /dev/microstrain)" />
           <arg name="imu_frame_id"  value="imu_link" />
-          <remap from="cv5/data"    to="imu/raw_raw" />
```

Comment:
I believe if you did "~data", this remap would work.

Reply:
I tried `~data`, `~/imu/data`, `imu/data`, `cv5/imu/data` and `/cv5/imu/data`.  None of them remapped the topic correctly.  I'm under the impression that the remap tag only works within `node`s and not within `include`s.